migrations/20190822070158-add-saved-column-to-payment-methods.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('PaymentMethods', 'saved', {
+      type: Sequelize.BOOLEAN,
+      defaultValue: false,
```

Comment:
If default is `false`, is there not a risk of not seeing all the other payment methods when they're created? I'm thinking about `virtualcard` and `collective` types in particular

Reply:
What would you recommend?

Reply:
Technically, I decided to only use this "saved" property for credit cards. For now.